ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -70,68 +71,84 @@
             <B.Tr>
               <B.Td>
                 <span class='hds-font-weight-semibold'>
-                  {{#if (can 'read model' B.data)}}
+                  {{#if (can 'read model' B.data.target)}}
                     <LinkTo
-                      data-test-visit-target={{B.data.id}}
+                      data-test-visit-target={{B.data.target.id}}
                       @route='scopes.scope.projects.targets.target'
-                      @model={{B.data.id}}
+                      @model={{B.data.target.id}}
                       @query={{hash isConnecting=false}}
                     >
-                      {{B.data.displayName}}
+                      {{B.data.target.displayName}}
                     </LinkTo>
                   {{else}}
-                    {{B.data.displayName}}
+                    {{B.data.target.displayName}}
                   {{/if}}
-                  {{#if B.data.description}}
-                    <p>{{B.data.description}}</p>
+                  {{#if B.data.target.description}}
+                    <p>{{B.data.target.description}}</p>
                   {{/if}}
                 </span>
               </B.Td>
               <B.Td>
                 <Copyable
-                  @text={{B.data.id}}
+                  @text={{B.data.target.id}}
                   @buttonText={{t 'actions.copy-to-clipboard'}}
                   @acknowledgeText={{t 'states.copied'}}
                 >
-                  <code>{{B.data.id}}</code>
+                  <code>{{B.data.target.id}}</code>
                 </Copyable>
               </B.Td>
               <B.Td>
-                {{#if B.data.type}}
+                {{#if B.data.sessions.length}}
```

Comment:
So we don't display anything if there's not an active session? How come we don't even display a no?